docs/Installation/Installation_Guide.rst

```diff
@@ -492,29 +492,29 @@ parameters as appropriate.)
    ::
 
     ingress:
-     enabled: "true"
-     annotations:
-       nginx.ingress.kubernetes.io/proxy-body-size: 0
-
-    hosts:
-      -  zenko.local
+      enabled: "true"
+      annotations:
+        nginx.ingress.kubernetes.io/proxy-body-size: 0
+      hosts:
+        - zenko.local
 
     cloudserver:
-     endpoint: "zenko.local"
+      endpoint: "zenko.local"
 
    You can edit these parameters, using each component’s values.yaml file
    as your guide. Save this file.
 
 2. If your Zenko instance is behind a proxy, append the following
-   lines to the options.yml file, substituting your proxy’s IP addresses
-   and port assignments:
+   lines to the options.yml file, under the section **cloudserver**, substituting
+   your proxy’s IP addresses and port assignments:
 
    ::
 
       proxy:

```

Comment:
Should we indent "proxy" as well for all the people who will copy paste this into their config?

Reply:
@ssalaues It's already indented. Or do you mean this?
```yaml
cloudserver:
  proxy:
    ...
```

Reply:
No just as in indenting the proxy settings so it can be copy pasted into the options.yaml file and work (which is likely what many will do)

Reply:
I think that's how it's done. It's extra indented so when you paste it falls under cloudserver?

Reply:
It's not extra indented.

Reply:
In the source it was, Github is not rendering the extra spaces.